mlflow/server/auth/__init__.py

```diff
@@ -8,15 +8,60 @@
 """
 
 import logging
+import uuid
+import os
+from pathlib import Path
+from flask import Flask, request, make_response, Response, redirect, flash, render_template_string
 
 from mlflow.server import app
+from mlflow.server.auth.config import read_auth_config
+from mlflow.server.auth.sqlalchemy_store import SqlAlchemyStore
+from mlflow.server.handlers import _get_rest_path
 
+_AUTH_CONFIG_PATH_ENV_VAR = "MLFLOW_AUTH_CONFIG_PATH"
 
 _logger = logging.getLogger(__name__)
 
+store = SqlAlchemyStore()
+
+
+class ROUTES:
+    USERS = _get_rest_path("/mlflow/users")
+    SIGNUP = "/signup"
```

Comment:
https://user-images.githubusercontent.com/17039389/230551196-21e844c9-52cc-46a5-bc64-014cbefc955c.mov

I'm testing sing-up. It it possible to redirect to `/` after sign-up?

Reply:
sure :)